src/protocol/gossip/impl/gossip_core.cpp

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <libp2p/protocol/gossip/impl/gossip_core.hpp>
+
+#include <cassert>
+
+#include <libp2p/protocol/gossip/impl/connectivity.hpp>
+#include <libp2p/protocol/gossip/impl/local_subscriptions.hpp>
+#include <libp2p/protocol/gossip/impl/message_builder.hpp>
+#include <libp2p/protocol/gossip/impl/remote_subscriptions.hpp>
+
+namespace libp2p::protocol::gossip {
+
+  // clang-format off
```

Comment:
why turn it off here?

Reply:
to pervent clang-format obfuscation of lambdas

Reply:
you do that too much in this file